frontend/invites/inviteInstitutionController.js

```diff
@@ -164,10 +164,7 @@
 
         function isSuperUser() {
             var current_institution_key = inviteInstCtrl.user.current_institution.key;
-            if (inviteInstCtrl.user.permissions.analyze_request_inst) {
-                return inviteInstCtrl.user.permissions.analyze_request_inst[current_institution_key]; 
-            }
-            return false;
+            return inviteInstCtrl.user.hasPermission('analyze_request_inst', current_institution_key);
```

Comment:
The current institution key can be retrieved inside the has permission function.

Reply:
A ideia foi criar um método genérico com o qual pudesse verificar qualquer tipo de permissão e não só a de instituição.